skimage/measure/_regionprops.py

```diff
@@ -819,6 +824,22 @@ def regionprops(label_image, intensity_image=None, cache=True):
     if not np.issubdtype(label_image.dtype, np.integer):
         raise TypeError('Label image must be of integer type.')
 
+    if coordinates is not None:
+        if coordinates == 'rc':
+            msg = ('The coordinates keyword argument to skimage.measure.'
+                   'regionprops is deprecated. All features are now computed '
+                   'in rc (row-column) coordinates. Please remove '
+                   '`coordinates="rc"` from all calls to regionprops before '
+                   'updating scikit-image.')
+            warn(msg, stacklevel=2)
```

Comment:
```suggestion
            warn(msg, stacklevel=2, category=DeprecationWarning)
```

Reply:
Might be worth updating the example in the contribution guide with the parameters `stacklevel` and `category` as well...

Reply:
`DeprecationWarning` are siliced if things are not invoked from `main` directly. I'm not sure that is what we want. Maybe `FutureWarning`

Reply:
Agree, after [looking that up](https://docs.python.org/3.7/library/exceptions.html#FutureWarning) `FutureWarning` (for end users) seems like a better option than `DeprecationWarning` (for developers).

Reply:
I've made this a FutureWarning.